src/main/scala/firrtl_interpreter/InterpretiveTester.scala

```diff
@@ -122,17 +122,17 @@ class InterpretiveTester(input: String, optionsManager: HasInterpreterSuite = ne
     * Pokes value to the named memory at offset
     *
     * @param name  the name of a memory
-    * @param offset the offset in the memory
+    * @param index the offset in the memory
```

Comment:
Why the change from `offset` to `index`? Won't this break existing code that uses named parameters? What are the units (bytes, words, ...)?

Reply:
Actually, not requesting changes so much as clarification.

Reply:
I just introduced the offset/index param in the previous PR, offset comes from the PeekPoke testers pokeAt where it makes more sense for a generic poke, but less sense than index for a memPoke.  I prefer the index, unless you really prefer offset.

Reply:
Index is fine.